### ISSUE TITLE
chdir not working

### ISSUE THREAD

**Issue by balq60** [terraform-linters/tflint#1709]
### Summary

I have configured the following in GitLab .pre-commit-config.yaml file

- repo: https://github.com/gruntwork-io/pre-commit
  rev: v0.1.18
  hooks:
    - id: tflint
      args: ["--chdir=IaC/environments/2-staging"]

I have put the following un-used variable in the following directory:

IaC/environments/1-test

variable "my_test" {
   description = "trip tflint"
   type        = string
   default     = null
}

When setting the args of:

      args: ["--chdir=IaC/environments/2-staging"]

I would expect the un-declared variable NOT to be reported, but it was:

tflint...................................................................Failed
- hook id: tflint
- exit code: 2

1 issue(s) found:

Warning: variable "my_test" is declared but not used (terraform_unused_declarations)

  on IaC/environments/1-test/compute-priv-variables.tf line 183:
 183: variable "my_test" {

Reference: https://github.com/terraform-linters/tflint-ruleset-terraform/blob/v0.2.2/docs/rules/terraform_unused_declarations.md

### Command

tflint --chdir=IaC/environments/2-staging

### Terraform Configuration

```terraform
Added this un-used variable

variable "my_test" {
   description = "trip tflint"
   type        = string
   default     = null
}
```

### TFLint Configuration

```HCL
No .hcl file used
```

### Output

```Shell
tflint...................................................................Failed
- hook id: tflint
- exit code: 2

1 issue(s) found:

Warning: variable "my_test" is declared but not used (terraform_unused_declarations)

  on IaC/environments/1-test/compute-priv-variables.tf line 183:
 183: variable "my_test" {

Reference: https://github.com/terraform-linters/tflint-ruleset-terraform/blob/v0.2.2/docs/rules/terraform_unused_declarations.md
```

### TFLint Version

0.45.0

### Terraform Version

v1.3.9

### Operating System

- [X] Linux
- [ ] macOS
- [ ] Windows

**Comment by wata727** [comment]
This looks like a pre-commit issue. Please report the issue there.
